app/assets/javascripts/map.js.coffee

```diff
@@ -85,9 +97,12 @@ App.Map =
       content = "<a href='/budgets/#{data["budget_id"]}/investments/#{data["investment_id"]}'>#{data["investment_title"]}</a>"
       return content
 
+    getProposalPopupContent = (url, title) ->
+      return "<a href='#{url}'>#{title}</a>"
```

Comment:
I think this change made more sense in the same commit which added the function :thinking:.